src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -327,6 +327,24 @@
         ),
         allow_gaps=False,
     ),
+    RetainKeywordSegment=StringParser(
+        "RETAIN",
+        KeywordSegment,
+        name="retain",
+        type="retain_keyword",
+    ),
+    DryKeywordSegment=StringParser(
+        "DRY",
+        KeywordSegment,
+        name="dry",
+        type="dry_keyword",
+    ),
+    RunKeywordSegment=StringParser(
+        "RUN",
+        KeywordSegment,
+        name="run",
+        type="run_keyword",
+    ),
```

Comment:
Why do we need these? Rather than just add to sparksql_keywords file?

Reply:
will update and move to the sparksql_keywords file